virtualmachine/vsphere/vm.go

```diff
@@ -471,6 +476,8 @@ type Destination struct {
 	// HostSystem specifies the name of the host to run the VM on. DestinationType ESXi
 	// will have one host system. A cluster will have more than one,
 	HostSystem string
+	// MorefID of managed object
+	MOID string `json:"MOID"`
```

Comment:
As currently MOID is only applicable in case of destination.Type ResourcePool, its better to mention that in comment, until we support it for other types also.